docs/proposals/label-associated-quota.md

```diff
@@ -0,0 +1,72 @@
+# Label Constrained Quota
+
+## Problem
+Cluster-administrators want to allot a total quota cap to a particular entity (users to start with) and allow that entity to allocate the quota amongst "his" projects as he sees fit.
+Our mechanism for relating entities is label selection, so we can create a `ResourceQuotaAllocation` object at the cluster scope that applies allotment totals against `ResourceQuota`
+objects.  By associating projects to owners using structured labels, it becomes possible to allocate quota the projects "owned" by a given user.
+
+
+## Types
+```go
+type ResourceQuotaAllocation struct{
+	unversioned.TypeMeta
+	kapi.ObjectMeta
+
+	Spec ResourceQuotaAllocationSpec
+	Status ResourceQuotaAllocationStatus
+}
+
+type ResourceQuotaAllocationSpec struct{
+	// Selector is used to find the set of projects that this ResourceQuotaAllocation is for
+	Selector extensions.LabelSelector
+
+	// Hard is the description of the limits
+	Hard kapi.ResourceQuotaSpec
+
+	// ResourceQuotaName is the name of the ResourceQuota document that will be updated when a project-owner attempts to allocate the quota in his project.
+	// This is important because it controls which ResourceQuota documents a user can modify.  These should roughly correspond to the scopes you've chosen
+	// for `Hard`.  That way these names will naturally overlap between overlapping ResourceQuotaAllocations.
+	// This name does **not** have any impact on accounting.  The max of any quota'd resource is the value used for enforcment
+	ResourceQuotaName string
```

Comment:
I am a little unclear on what this means.   This is the pet name of a quota that the user is or is not allowed to edit?